openwisp_controller/config/base/device.py

```diff
@@ -335,7 +337,7 @@ def _get_initial_values_for_checked_fields(self):
         if not present_values:
             return
         self.refresh_from_db(fields=present_values.keys())
-        for field in self._changed_checked_fields:
+        for field in present_values:
```

Comment:
Forgot to add a comment here, will add it in next commit : 

Let me know if I am wrong, but we are adding fields in `present_fields` based on certain conditions, so shouldn't we loop in `present_fields` instead of `_changed_checked_fields`?

Reply:
Not sure on this but it should work